source/simulator/solver_schemes.cc

```diff
@@ -573,19 +573,67 @@ namespace aspect
               {
                 const bool EisenstatWalkerChoiceOne = true;
                 parameters.linear_stokes_solver_tolerance = compute_Eisenstat_Walker_linear_tolerance(EisenstatWalkerChoiceOne,
-                                                            parameters.maximum_linear_stokes_solver_tolerance,
+                                                            newton_handler->get_maximum_linear_stokes_solver_tolerance(),
                                                             parameters.linear_stokes_solver_tolerance,
                                                             stokes_residual,
                                                             residual,
                                                             residual_old);
 
-                pcout << "   The linear solver tolerance is set to " << parameters.linear_stokes_solver_tolerance << std::endl;
+                const std::string preconditioner_stabilization = newton_handler->get_preconditioner_stabilization_string(newton_handler->get_preconditioner_stabilization());
+                const std::string velocity_block_stabilization = newton_handler->get_velocity_block_stabilization_string(newton_handler->get_velocity_block_stabilization());
+                pcout << "   The linear solver tolerance is set to " << parameters.linear_stokes_solver_tolerance << ". Stabilisation Preconditioner is " << preconditioner_stabilization << " and A block is " << velocity_block_stabilization << "." << std::endl;
```

Comment:
-> stabilization (with a z)

I'd also show the value for the *matrix* first, before the value for the preconditioner

Reply:
Why do you think that that order is more logical? In my mind the preconditioner comes first.

Reply:
I don't know -- I think as preconditioners as an ancillary tool to solving the linear system.

I don't feel strongly about the issue if you do :-)